### PR TITLE
update branding and use local docker images

### PR DESCRIPTION
**Pull request by joshtab**
TODO: use tags for docker images

### CODE REVIEW

**Comment by nichochar** [comment]
Is this updated branding also going to affect the project name?

File: docker-compose.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ services:
 
     snappass:
         build: .
-        image: pinterest/snappass
+        image: "09bcea61bedc"
```
Comment (nichochar) [inline docker-compose.yml]: @jparise just checking this is expected

**Comment by joshtab** [comment]
Please disregard this PR

This PR was an attempt to push changes to my company's internal fork of snappass (thank you for creating and open sourcing it), but I selected the wrong target within github

**Comment by nichochar** [comment]
Oh my bad, I misread the github UI and thought it got merged, not deleted. All good :+1:

**Pull request closed without merging**